File: WeekOne_C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int pyHeight;

    //get the pyramid's height from the user
    //will keeping ask until a valid number, between 1 and 8 (inclusives)
    do
    {
        pyHeight = get_int("How tall is the pyramid? ");
    }
    while (pyHeight < 1 || pyHeight > 8);

    //for each row
    for (int i = 1; i <= pyHeight; i++)
    {
        int aux = i;

        //if it's not the last row, add blank spaces to the left
        while ((pyHeight - aux) > 0)
        {
            printf(" ");
            aux++;
        }

        //each column
        //left half
        for (int j = i; j > 0; j--)
        {
            printf("#");
        }

        //spaces to separate halves
        printf("  ");

        //right half
        for (int k = i; k > 0; k--)
        {
            printf("#");
        }

        //jump to new row
        printf("\n");
    }
}

File: week6-Python/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height >= 1 and height <= 8:
        break

hashes = 1
for i in range(height, 0, -1):
    print(" " * (i - 1), end="")
    print("#" * hashes, end="")

    print("  ", end="")

    print("#" * hashes)

    hashes += 1